docs/conf.py

```diff
@@ -59,7 +59,7 @@
 #
 
 # The full version, including alpha/beta/rc tags.
-release = '1.15.5'
+release = '1.16'
```

Comment:
Can you remove the commits that change the version string? This is something that's done at release time.